src/renderer/components/Text.tsx

```diff
@@ -0,0 +1,77 @@
+import * as React from 'react'
+import { PropsWithChildren } from 'react'
+import Typography, { TypographyProps } from '@mui/material/Typography'
+
+type BaseProps = PropsWithChildren &
+  TypographyProps & { style?: React.CSSProperties }
+
+export function TitleText({ children, style, ...otherProps }: BaseProps) {
+  return (
+    <Typography variant="h1" style={style} {...otherProps}>
+      {children}
+    </Typography>
+  )
+}
+
+export function BodyText({ children, style, ...otherProps }: BaseProps) {
+  return (
+    <Typography variant="body1" style={style} {...otherProps}>
+      {children}
+    </Typography>
+  )
+}
+
+export function DescriptionText({ children, style, ...otherProps }: BaseProps) {
+  return (
+    <Typography variant="body2" style={style} {...otherProps}>
+      {children}
+    </Typography>
+  )
+}
+
+export function SubtitleText({ children, style, ...otherProps }: BaseProps) {
+  return (
+    <Typography variant="subtitle1" style={style} {...otherProps}>
+      {children}
+    </Typography>
+  )
+}
+
+type TextProps = BaseProps & {
+  kind?: 'title' | 'body' | 'description' | 'subtitle'
```

Comment:
think `description` is kind of vague, especially given `body` (when do i know which to use?). maybe it can be omitted for now and reintroduced with some clearer name if needed?